src/vs/workbench/api/node/extHostTunnelService.ts

```diff
@@ -237,16 +250,18 @@ export class ExtHostTunnelService extends Disposable implements IExtHostTunnelSe
 			});
 		}));
 
-		const procChildren = fs.readdirSync('/proc');
-		const processes: { pid: number, cwd: string, cmd: string }[] = [];
+		const procChildren = await pfs.readdir('/proc');
+		const processes: {
+			pid: number, cwd: string, cmd: string
+		}[] = [];
 		for (let childName of procChildren) {
 			try {
 				const pid: number = Number(childName);
 				const childUri = resources.joinPath(URI.file('/proc'), childName);
-				const childStat = fs.statSync(childUri.fsPath);
+				const childStat = await pfs.stat(childUri.fsPath);
 				if (childStat.isDirectory() && !isNaN(pid)) {
-					const cwd = fs.readlinkSync(resources.joinPath(childUri, 'cwd').fsPath);
-					const cmd = fs.readFileSync(resources.joinPath(childUri, 'cmdline').fsPath, 'utf8');
+					const cwd = await promisify(fs.readlink)(resources.joinPath(childUri, 'cwd').fsPath);
```

Comment:
By the way, since Node 12 there's a `promises` export from `fs`, I usually do `import { promises as fs } from 'fs'` 🙂